tabbycat/importer/views.py

```diff
@@ -221,18 +221,18 @@ def post(self, request, *args, **kwargs):
             management.call_command(importtournament.Command(), source,
                                     force=True, strict=False, encoding='utf-8')
         except TournamentDataImporterError as e:
-            messages.error(self.request, mark_safe(
+            messages.error(self.request, mark_safe(_(
                 "<p>There were one or more errors creating the demo tournament. "
                 "Before retrying, please delete the existing demo tournament "
                 "<strong>and</strong> the institutions in the Edit Database Area.</p>"
                 "<p><i>Technical information: The errors are as follows:"
-                "<ul>" + "".join("<li>{}</li>".format(message) for message in e.itermessages()) + "</ul></i></p>"
+                "<ul>") + "".join("<li>{}</li>".format(message) for message in e.itermessages()) + "</ul></i></p>"
```

Comment:
I think it might be better for the end of the translatable string to go before the `<ul>` tag? I'm not sure what Crowdin does but some translation platforms try to be intelligent about matching HTML tags, and having just a single one could confuse both machines and humans. (Thoughts?)

Reply:
I prefer to match. Crowdin might complain for unmatched tags but doesn't try to be smart about it, so it is less of a concern.

Reply:
It doesn't really, I think it just makes it look like there was a mistake. If it's important to indicate to translators, it should be in a context note. (But I think in this case it's clear from the English, so no need.)